### PR TITLE
Add Catalan translation

### PR DESCRIPTION
**Pull request by sheilagomes**
Add Catalan translations and fix crowdin configuration file. Tracked in task [LOC-9754](https://vtex-dev.atlassian.net/browse/LOC-9754).

[LOC-9754]: https://vtex-dev.atlassian.net/browse/LOC-9754?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: dist/self-service-reasons/1.12.0/cancellation-reasons_ca.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
+{
+  "name": "Formulari de cancel·lació",
+  "locale": "cat",
```
Comment (camilavcoutinho) [inline dist/self-service-reasons/1.12.0/cancellation-reasons_ca.json:3]: é 'cat' mesmo?

File: dist/self-service-reasons/1.12.0/replacement-reasons_ca.json [hunk 2]
```diff
@@ -0,0 +1,94 @@
+{
+  "name": "sacForm",
+  "locale": "cat",
```
Comment (camilavcoutinho) [inline dist/self-service-reasons/1.12.0/replacement-reasons_ca.json:3]: aqui também

File: dist/self-service-reasons/1.12.0/cancellation-reasons_ca.json (hunk 1 above)
Comment (sheilagomes) [reply dist/self-service-reasons/1.12.0/cancellation-reasons_ca.json]: Corrigido, obrigada!

File: dist/self-service-reasons/1.12.0/replacement-reasons_ca.json (hunk 2 above)
Comment (sheilagomes) [reply dist/self-service-reasons/1.12.0/replacement-reasons_ca.json]: Corrigido

**Review by camilavcoutinho — APPROVED** [review]

**Review by lucasecdb — APPROVED** [review]

**Pull request merged**